python/hsfs/training_dataset.py

```diff
@@ -170,22 +170,20 @@ def feed(
         cycle_length=2,
     ):
         """
-        :param target_name: name of the target variable
-        :type target_name: str, required
-        :param split: training dataset split name, train, test or eval, defaults to None
-        :type split: str, optional
-        :param feature_names: name of training variables, defaults to None
-        :type feature_names: 1d array, optional
-        :param is_training:  whether it is for training, testing or eval, defaults to True
-        :type  is_training: boolean, optional
-        :param cycle_length: number of files to be read and deserialized in parallel, defoults to 2
-        :type cycle_length: int, optional
-        :param engine: execution engine. defaults  to spark
-        :type engine: str, optional
-        :return: feed model engine object
-        :rtype: FeedModelEngine
+        tf_data function returns TFDataEngine object that has utility methods for tf.data
+
+        # Arguments
+            training_dataset: `str`: training dataset name
+            split: split: `Optional[str]`: name of training dataset split. train, test or eval
```

Comment:
same as above, use python type in the function signature, then you don't need to write them manually here.